events/schema.py

```diff
@@ -471,6 +471,28 @@ def resolve_reference_id(self, info, **kwargs):
         return self.reference_id
 
 
+class TicketmasterEnrolmentNode(DjangoObjectType):
```

Comment:
Would user view restrictions like in `EnrolmentNode.get_queryset` function need to be implemented for TicketmasterEnrolmentNode?

Reply:
Good point.

In this particular case it shouldn't be strictly necessary yet, as those Ticketmaster enrolments can be accessed only through a child node, and that requires permissions. However, in my experience it is pretty easy to expose data in Graphene unintentionally, so I guess it is better to be safe than sorry and implement the restriction to the node's queryset already. That will be also consistent with the other nodes. The downside of course is that the resulting DB query will be a bit more complex, but I don't think that will be an issue in practice. 

So yeah, I will make that addition 🙂